.github/workflows/engine_vdscanner_testtools_delivery.yml

```
@@ -0,0 +1,76 @@
+name: Engine VD Scanner Test Tools Delivery
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on pull request but only changes in src/engine/source/vdscanner/testtools/ directory.
+  pull_request:
+    types: [synchronize, opened, reopened, ready_for_review]
```

We want the trigger to be when we merge a PR into master
```suggestion
  pull_request:
    branches: master
    types: closed
```

---

Added, although I think it would be useful to have also on PRs, if we make changes we need to know it compiles without waiting until is merged. 
Also, what would happen when a PRs is merged against 5.0.0, or 6.0.0, etc? isn't "branches: master" too rigid?

---

> Also, what would happen when a PRs is merged against 5.0.0, or 6.0.0, etc? isn't "branches: master" too rigid?

Yes, we should think of something for the future

> Added, although I think it would be useful to have also on PRs, if we make changes we need to know it compiles without waiting until is merged.

This is true, we could compile the tools without uploading the artifacts